src/openforms/js/components/admin/form_design/registrations/json_dump/JSONDumpVariableConfigurationEditor.js

```diff
@@ -1,22 +1,32 @@
 import {useFormikContext} from 'formik';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {useContext} from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import {FormContext} from 'components/admin/form_design/Context';
 import {VARIABLE_SOURCES} from 'components/admin/form_design/variables/constants';
 import {getVariableSource} from 'components/admin/form_design/variables/utils';
 import Field from 'components/admin/forms/Field';
 import FormRow from 'components/admin/forms/FormRow';
 import {Checkbox} from 'components/admin/forms/Inputs';
 
+import {CUSTOM_COMPONENTS_SCHEMA} from '../utils';
+
 const JSONDumpVariableConfigurationEditor = ({variable}) => {
   const {
-    values: {variables = [], additionalMetadataVariables = [], fixedMetadataVariables = []},
+    values: {
+      variables = [],
+      additionalMetadataVariables = [],
+      fixedMetadataVariables = [],
+      transformToList = {},
```

Comment:
```suggestion
      transformToList = {},
```
would work with a list here instead of an object